source/_integrations/picnic.markdown

```diff
@@ -24,7 +24,7 @@ This integration provides the following sensors. Some sensors are disabled by de
 
 | Name                           | Description                                                                                                                                         |
 | ------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Cart items count               | The amount of different products currently in the cart.                                                                                             |
+| Picnic shopping cart           | The amount of different products currently in the cart.                                                                                             |
```

Comment:
```suggestion
| Picnic shopping cart           | The number of different products currently in the cart.                                                                                             |
```
*different products*: does this mean unique by type? or just the number of products in total? For example, if I have 5 apples and 2 cakes, would these count as 2 different products? or 7?